news/2023/2023-11-07-winter-fanart-contest.md

```diff
@@ -0,0 +1,53 @@
+
+If your answer to this question is yes, then pull up a chair and prepare yourselves for a brand-new food-related theme! **Draw any of the osu! mascots (or community mascots) enjoying a seasonal feast** for fun and glory!
+
+You may be wondering what the heck a *seasonal feast* is. Since we're a massively multicultural community, there's a ton of traditional festivals and feasts that roughly align around the end of the year, from anything like Thanksgiving, Diwali/Deepavali, Hanukkah, the winter solstice (including the Dongzhi festival), Festivus, Christmas, Kwanzaa, and tons more that we could probably spend a whole article describing.
+
+Suffice to say, the end of the year involves a *lot* of celebrating and eating, so we figured this year, we'd try trussing up the idea and seeing what delectable creations the immensely talented artists of the osu! community can cook up. *Shokugeki no Soma* enthusiasts who've wanted to draw glazed ham (and we know there is at LEAST *one* of you), rejoice!
+
+If you're looking for ideas, here's a few to get the juices flowing:
+
+- The mascots fighting over who gets the last piece of Christmas ham
+- Mocha rolling a comically overstuffed don around in front of a fireplace
+- Yuzu being spear-tackled by Chirou right as he was about to finish his fruit pyramid centrepiece
+- Mari pulverising Mani with a turkey drumstick for the sin of being an annoying younger sibling
+- ...and basically anything you can think of that involves a seasonal, end-of-year feast!
+
+This is a pretty broad theme, so feel free to go wild with it — just make sure it stays vaguely festive. If you want snow, go for it! If you don't, that's fine too. As long as the mascots are enjoying their time together in the company of good friends, we'll consider that fair game.
+
+You've got **21 days** to bring your creation to life, so time to get cracking on the crackling! As always, a selection of the top voted finalists will make it to the main menu and receive 3 months of osu!supporter, plus the top 3 will also receive fancy bespoke profile badges pulled from their own creations.
+
+But make sure you read the contest rules below first, especially because there's a new one this time around.
+
+## Rules
+
+- **By submitting your artwork to this contest, you grant us permission to use your artwork in the main menu client should it win, and to use it in promotional material for both the game (osu!) and the contest (Winter 2023 Fanart Contest).** You retain all rights to your creation and you can sell it and distribute it however you like — it is your work, after all. We're just saying here that if you submit your piece to this contest, you should be okay with us using it.
+- **All entries MUST follow the contest theme (Winter/end-of-year seasonal feast) and contain AT LEAST one of the osu! official or community mascots.** The mascots are not optional and neither is the feast/festival/celebration!
```

Comment:
`feast/festival/celebration` makes it sounds as if food does not actually need to be involved, but from reading the newspost it seems to be the actual focus?

Reply:
considering the amount this is emphasised in the post, with the prompt being "Draw any of the osu! mascots (or community mascots) enjoying a seasonal feast" and the theme being "Winter/end-of-year seasonal feast", i don't think there's a whole lot of room for misunderstanding

Reply:
```suggestion
- **All entries MUST follow the contest theme (Winter/end-of-year seasonal feast) and contain AT LEAST one of the osu! official or community mascots.** The mascots are not optional and neither are the feasts and festivities!
```